src/bot/MessageHandler.ts

```diff
@@ -26,6 +26,9 @@ export default class MessageHandler {
   }
 
   private isValidMessage(message: Message) {
+    if(this.config.admin && this.config.admin !== '') {
+      if(message.author.id !== this.config.admin) return false;
```

Comment:
1. Remove the nested conditions: `if (this.config.mainAdminId && this.config.mainAdminId !== message.author.id)`
2. This does not really change the validity of a message. Extract this into another method `isAdminRequired()` and use like this

```javascript
public handle(message: Message) {
  if (this.isAdminRequired()) return;
  if (!this.isValidMessage(message)) return;
  // ...
}
```